snapper-rollback.py

```diff
@@ -124,6 +158,22 @@ def rollback(subvol_main, subvol_main_newname, subvol_rollback_src, dev, dry_run
             else:
                 os.rename(subvol_main_newname, subvol_main)
 
+def getNextSubvolumeNumber(mountpoint,config,dry_run=False):
+  if dry_run: 
+    subvol_main_snapshot_number = str(int(os.listdir(path= "/.snapshots")[-1])+1)
+  else:
+    subvol_main_snapshot_number = str(int(os.listdir(path= 
+                                               mountpoint / config.get("root", "subvol_snapshots")
+                                               )[-1])+1
```

Comment:
I think this should be the same whether dry-run or not.
Also, beware files not being listed in alphanumerical order. This should be sorted